src/views/product-landing/components/heading/index.tsx

```diff
@@ -0,0 +1,24 @@
+import { HeadingProps } from './types'
+import clamp from 'lib/clamp'
+import s from './style.module.css'
+
+function Heading({
+  slug,
+  heading,
+  level,
+  size,
```

Comment:
Can we talk about this `size` prop a little more? At the moment, I feel like it enables too much mixing and matching of heading elements and heading font styles. I personally think that if someone wants to mix and match styles, they should have to really work for it via sending in a `className` -- I think a manual approach can help ensure someone is doing something with well thought out intentions. I do realize the trade off with that approach is that a ton of things become adjustable away from the styles specified by design systems.

Reply:
Caught up a bit on the Slack chatter on this - my impression so far is we need something like this `size` prop to implement current designs without breaking HTML semantics (since in current design, the `Explore Vault Documentation` feels semantically like an `h2` but is styled with `heading-1`, which the remainder of the headings on the page also feel semantically like `h2`s and are styled with `heading-2`).

VERY on board with your suggestion of changing the style names so they don't sound exactly like HTML elements 🙌 💯 even the short description above makes my brain hurt a lil.

Reply:
I think we do need this for now, and if they go with your suggestion of renaming to use `100`-ish names, then this `size` prop is probably the appropriate naming. Happy for this to be left as is for now since we do need _something_ for the time being to allow mixing and matching.